### PR TITLE
Fixed Interpolated Distribution's interval transform initialization

### PR DESCRIPTION
**Pull request by kc611**
Linked Issue: https://github.com/pymc-devs/pymc/issues/5048

### CODE REVIEW

**Review by ricardoV94 — CHANGES_REQUESTED** [review]
Looks good, but we should add a regression test

File: pymc/tests/test_distributions.py [hunk 1]
```diff
@@ -2676,6 +2676,13 @@ def ref_pdf(value):
 
                 self.check_logp(TestedInterpolated, R, {}, ref_pdf)
 
+    def test_interpolated_inits(self):
+        # Issue: https://github.com/pymc-devs/pymc/issues/5048
+        x_points = np.linspace(0, 10, 100000)
+        pdf_points = sp.norm.pdf(x_points, loc=1, scale=1)
+        with pm.Model() as m:
+            x = pm.Interpolated("interpolated", x_points, pdf_points)
```
Comment (ricardoV94) [inline pymc/tests/test_distributions.py]: We should assert something, even though this was failing before the PR by itself. Perhaps eval the logp outside of the range and check it's not -inf?

**Review by ricardoV94 — CHANGES_REQUESTED** [review]

**Review by ricardoV94 — APPROVED** [review]

**Comment by ricardoV94** [comment]
@kc611 I updated the PR. Let me know if you see any problems

**Review by kc611 — COMMENTED** [review]
Looks Good. Back then I was confused as of what tests to add. This looks pretty reasonable.

**Pull request merged**